locales/en-US/editor.properties

```diff
@@ -125,7 +125,7 @@ CMD_TOGGLE_QUICK_DOCS=Quick Docs
 
 # Drag and Drop
 
-DND_MAX_FILE_SIZE_EXCEEDED=file exceeds maximum supported size: 3MB
+DND_MAX_SIZE_EXCEEDED=file exceeds maximum supported size: {0} MB.
```

Comment:
technically no. You can just have it in Thimble and it will automatically come into this repo